pkg/arvo/app/publish.hoon

```diff
@@ -1005,14 +951,161 @@
       (generate-invites book (~(del in invitees.group) our.bol))
   ==
 ::
+++  handle-poke-fail
+  |=  wir=wire
+  ^-  (quip card _state)
+  ?+  wir
+    [~ state]
+  ::  new note failed, stash it in limbo
+  ::
+      [%forward %new-note @ @ @ ~]
+    =/  host=@p  (slav %p i.t.t.wir)
+    =/  book-name  i.t.t.t.wir
+    =/  note-name  i.t.t.t.t.wir
+    =/  book  (~(get by books) [host book-name])
+    ?~  book
+      [~ state]
+    =/  note  (~(get by notes.u.book) note-name)
+    ?~  note
+      [~ state]
+    =.  notes.limbo   (~(put by notes.limbo) [host book-name note-name] u.note)
```

Comment:
How is the limbo state rendered? Is it cleared regularly? Is it retried?

Reply:
It's not rendered anywhere. In the case of event failures, edits and deletes remove the thing stored in limbo and put it in our regular state. Failed adds remove the thing from our regular state and put it in limbo. Currently there's no way non-involved way of getting it back out of limbo, but I figured it would be better to at least store it and give savvy users that option than just swallowing it.

Reply:
Hmm. Okay. Could you add a TODO relating to cleaning them up somehow or presenting them in some "weird limbo pending" interface later?

Reply:
> [Limbo state is] not rendered anywhere.

Huh. Wasn't that the whole point of this? If you're _just_ storing it locally, and not rendering it, that doesn't do anything to improve the UX.

Reply:
@loganallenc 
Maybe I should just add a rudimentary poke %noun and scry interface for managing this. Probably better than leaving it.

@Fang- 
We use the normal non-limbo structure for pending posts/comments, in order to reuse the read endpoint code, rather than have to rewrite all of that. Limbo is for storing data that allows us to recover from failures that happen in a pending state.

Reply:
Hmm, interesting. My naively thought up implementation here was to store pending posts/comments separately, as you are doing, but then include them in the already-existing scry/subscription endpoint results by simply welding them on top of the "confirmed" lists.

Reply:
That would be a more invasive change, requiring a bunch of library functions and reducer logic to change.